MekHQ/src/mekhq/campaign/parts/equipment/AmmoBin.java

```diff
@@ -552,6 +552,8 @@ public static void swapAmmoFromCompatible(Campaign campaign, int needed, AmmoSto
         AmmoType aType;
         AmmoType curType = (AmmoType)as.getType();
         int converted = 0;
+
+        // TODO: make this less intensive.
         for(Part part : campaign.getSpareParts()) {
```

Comment:
Was this supposed to be swapped over?

Reply:
No it has side effects and needs a different solution.